Source/SPAutosizingTextView.h

```diff
@@ -0,0 +1,17 @@
+//
+//  SPAutosizingTextView.h
+//  Sequel Ace
+//
+//  Created by Jason Morcos on 7/8/20.
+//  Copyright © 2020 Sequel-Ace. All rights reserved.
+//
+
+#import <Cocoa/Cocoa.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SPAutosizingTextView : NSTextView
```

Comment:
What about Swift version, so we start slowly, hmm? :))

Reply:
I guessss haha

Reply:
Swiftified! Take a look

Reply:
And also now ARC compatible 🎉